internal/compiler/typeregister.rs

```diff
@@ -84,13 +84,19 @@ pub(crate) const RESERVED_DROP_SHADOW_PROPERTIES: &[(&str, Type)] = &[
     ("drop-shadow-color", Type::Color),
 ];
 
+pub(crate) const RESERVED_ACCESSIBILITY_PROPERTIES: &[(&str, Type)] = &[
+    //("accessible-role", ...)
+    ("accessible-label", Type::String),
```

Comment:
Can you use accessible-name, please? That is what all the accessible toolkits call this.

Reply:
Flutter also seems to call it label https://api.flutter.dev/flutter/dart-html/AccessibleNode/label.html (i think that's what we're talking about)

Personally, I like "label" better than "name"